config/environments/development.rb

```diff
@@ -46,12 +54,18 @@
   config.assets.quiet = true
 
   # Raises error for missing translations.
-  # config.action_view.raise_on_missing_translations = true
+  # config.i18n.raise_on_missing_translations = true
```

Comment:
It'd be nice to enable this for the dev environment!

Reply:
That sounds like a good shout - how about the test environment too? That way if any of our tests trigger a view that is missing a translation, the test should fail.

Reply:
Sounds good. Adding this option to the test env can be troublesome if we dynamically generate locales (e.g. by document type) if the tests are using funky data, but as the suite has passed, this issue doesn't seem to be present in this app 👍